scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -225,6 +229,83 @@ def __init__(self, test: dict, config: dict, definitions: SpecDefinitions, pics_
                     continue
                 get_constraints(value['constraints'])
 
+    def _update_mappings(self, test: dict, definitions: SpecDefinitions):
+        cluster_name = self.cluster
+        if definitions is None or not definitions.has_cluster_by_name(cluster_name):
+            self.argument_mapping = None
+            self.response_mapping = None
+            self.response_mapping_name = None
+            return
+
+        argument_mapping = None
```

Comment:
Put this in front of the if condition above and make the if just return? Seems to avoid some duplication of setting everything to null.

Reply:
Please note that the lines above are doing `self.argument_mapping` versus `argument_mapping`.

Reply:
I have opened the followup here: https://github.com/project-chip/connectedhomeip/issues/25153